R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R

```diff
@@ -99,13 +99,13 @@ module_aglu_LA107.an_IMAGE_R_C_Sys_Fd_Y <- function(command, ...) {
     L107.an_Prod_Mt_ctry_C_Y %>%
       # bring in the mixed fraction for each country, commodity, year as value.y. A left_join rather than
       # left_join_error_no_match is used because we preserve the NA's from mismatches:
-      left_join( L100.IMAGE_an_Prodmixfrac_ctry_C_Y, by = c("iso","year", "GCAM_commodity"="commodity"))%>%
+      left_join( L100.IMAGE_an_Prodmixfrac_ctry_C_Y, by = c("iso", "year", "GCAM_commodity"="commodity"))%>%
       # mixed animal production = total animal production * fraction mixed for each country, commodity, year:
       mutate(value = value.x * value.y) %>%
       # drop columns don't care about:
       select(-value.x, -value.y, - IMAGE_region_ID) %>%
```

Comment:
Yeah, we had to fix the variable types on the incoming files. I'll remove the space.